theme-dev/MaterialYou.md

```diff
@@ -0,0 +1,9 @@
+<h1 align="center">Material You</h1>
+The Themer Plugin has custom Material You strings that allow you to make UI Elements match your devices Material You colors by pulling colors from Google's Material You API. This requires directly modifying the JSON file, we recommend coming into this with at least a little bit of experience to avoid screwing something up.
+
+### Getting Started
+To get started you need a device with an Android 12 ROM that includes support for Material You. I recommend installing [this app](https://play.google.com/store/apps/details?id=com.ch3d.material.color&hl=en_US&gl=US), this will help you later on. Open the Themer Plugin menu and create a theme, this creates the JSON file that you will use for adding Material You values. In the Themer plugin add random colors to all the UI Elements you'd like to change (this saves time later in the process). Now that you have that open the JSON file in your prefered text editor then move on to the next section.
```

Comment:
explain what the app is for. also please insert new lines to prevent lines from being too long, lines shouldn't be longer than 150 characters including spaces. single new lines are ignored in markdown

Reply:
I think I fixed this, let me know if I did it right.